rotkehlchen/tests/unit/test_tasks_manager.py

```diff
@@ -62,6 +62,7 @@ def fixture_task_manager(
         eth_transactions=eth_transactions,
         deactivate_premium=lambda: None,
         query_balances=lambda: None,
+        update_curve_pools_cache=lambda: None,
```

Comment:
No test to check that the periodic task is run? Just how it's being done in this module?

Reply:
Oh yeah, forgot about the periodic task test, will add it, thanks